token-service/src/main/java/org/p2p/token/service/model/TokenMetadataResult.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.token.service.model
+
+import org.p2p.core.token.TokenMetadata
+
+sealed interface TokenMetadataResult {
```

Comment:
UpdateTokenMetadataResult 
I can't see any action in the name that will be result of